lib/atat-web-api-stack.ts

```diff
@@ -55,24 +51,22 @@ export class AtatWebApiStack extends cdk.Stack {
       )
     );
 
-    // State Machine
+    // State Machine and workflow
     const provisioningSfn = new ProvisioningWorkflow(this, "ProvisioningWorkflow", { environmentName });
-    this.provisioningStateMachine = new StateMachine(this, "ProvisioningStateMachine", {
-      stateMachineProps: {
-        definition: provisioningSfn.workflow,
-      },
-      logGroup: provisioningSfn.logGroup,
-    }).stateMachine;
 
     // Provisioning lambda that translates and invokes the state machine
     const provisioningJob = new ApiSfnFunction(this, "ProvisioningJobRequest", {
       method: HttpMethod.POST,
```

Comment:
Have we talked about https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway-readme.html#aws-stepfunctions-backed-apis?

Reply:
ehh maybe more specifically https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway.StepFunctionsIntegration.html

Reply:
I don't recall previous discussions of StepFunctions backed APIs

Reply:
Yeah it was [mentioned previously](https://github.com/dod-ccpo/atat-web-api/pull/638#discussion_r826276454), and I thought a ticket was supposed to come out of it, so I didn't think about changing it.

Reply:
Thank you for the reminder.  I didn't connect the two.  I do have a note to create that ticket from #638, but haven't yet done so.

Reply:
Oh it's AT-7177!